torch/csrc/jit/graph_executor.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <memory>
+#include "torch/csrc/jit/ir.h"
+
+namespace torch { namespace jit {
+
+struct GraphExecutorImpl;
+struct GraphExecutor {
+  GraphExecutor() {}
+  GraphExecutor(std::shared_ptr<Graph> graph, bool optimize = true);
```

Comment:
If not specified, it is computed.

Reply:
Yeah I realized that later, because I reviewed the header first. Can you still add a comment? If I haven't seen the implementation I would assume it's just conservative and sets it to false